src/ddscxx/src/dds/core/policy/CorePolicy.cpp

```diff
@@ -42,6 +42,7 @@ OMG_DDS_DEFINE_POLICY_TRAITS(dds::core::policy::TopicData,           TopicData)
 OMG_DDS_DEFINE_POLICY_TRAITS(dds::core::policy::GroupData,           GroupData)
 OMG_DDS_DEFINE_POLICY_TRAITS(dds::core::policy::TransportPriority,   TransportPriority)
 OMG_DDS_DEFINE_POLICY_TRAITS(dds::core::policy::Lifespan,            Lifespan)
+OMG_DDS_DEFINE_POLICY_TRAITS(dds::core::policy::DataRepresentation,  DataRepresentation)
```

Comment:
Please also encapsulate this line with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro.

Reply:
Will do.